### ISSUE TITLE
Si lattice parameter for LDA

### ISSUE THREAD

**Issue by braun91** [shankar1729/jdftx#26]
Hi,

I was doing some test calculations for Si bulk and was a little bit surprised about the lattice parameter I obtained using internal lattice optimization as well as Birch-Murnaghan-equation.
I am using the GBRV pseudopotentials and oversampled the unit cell with 20x20x20 kpoints and cutoff energies of 40 and 200 hartree.
The obtained value of 5.433 Angstrom matches the experimental value of 5.431 Angstrom (see https://physics.nist.gov/cgi-bin/cuu/Value?asil) extremely well. However, for LDA I would expect a lattice parameter that is around 1% LOWER than the experimental one.
Can you comment on that? Has there been some "tuning" of internal parameters in the pseudopotential?

Thank you in advance!

**Comment by shankar1729** [comment]
That's quite interesting! As far as I remember, the GBRV pseudopotentials were tested / tuned against all-electron calculations, so they should not exhibit any special bias towards experimental results. Note that they optimized the pseudopotentials for PBE and then generated the LDA and PBEsol versions with the same parameters. My group has noticed slightly unexpected behaviour with some of the PBEsol pseudopotentials, so it is very possible that similar issues arise in the LDA ones.

In any case, this would be a question that the developers of the GBRV pseudopotentials (https://www.physics.rutgers.edu/gbrv/) can best answer.

Cheers,
Shankar

**Comment by braun91** [comment]
I just tried two different LDA pseudopotentials from Quantum Espresso (http://www.quantum-espresso.org/upf_files/Si.pz-n-rrkjus_psl.0.1.UPF & http://www.quantum-espresso.org/upf_files/Si.pz-vbc.UPF) and both also result in slightly too large large lattice parameter.
I tested this on two different machines.
Maybe this has to do with JDFTX?
I just got an email from the developer of GBRV saying the get the (correct) underestimated lattice constant of 5.40 Angstrom with QE.

Here are the parameters I used:

include Si_lda.lattice
include Si_lda.ionpos

kpoint-folding 20 20 20
ion-species GBRV/$ID_lda.uspp
elec-cutoff 40 200

electronic-SCF
lattice-minimize nIterations 100
dump-name Si_lda.$VAR
dump Ionic State
